pkgs/by-name/fl/flaresolverr/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  makeWrapper,
+
+  chromium,
+  python3,
+  undetected-chromedriver,
+  xorg,
+
+  nixosTests,
+}:
+
+let
+  python = python3.withPackages (
+    ps: with ps; [
+      bottle
+      func-timeout
+      prometheus-client
+      selenium
+      waitress
+      xvfbwrapper
+
+      # For `undetected_chromedriver`
+      looseversion
+      requests
+      websockets
```

Comment:
Is there a reason that we cannot include these dependencies with the actual `undetected-chromedriver` package?

Reply:
These are for the [altered version](https://github.com/FlareSolverr/FlareSolverr/tree/master/src/undetected_chromedriver) of the `undetected-chromedriver` package that the FlareSolverr folk maintain. The [original one](https://github.com/ultrafunkamsterdam/undetected-chromedriver/) packaged in this PR is used to create the `undetected-chromedriver` binary, which is then used by that altered version of the library.